cmake/Scripts/ExecuteTests.cmake

```diff
@@ -0,0 +1,11 @@
+include(${LOCATIONS_FILE})
+
+string(REPLACE " " ";" EXE_IUT_LIST "${IUT_LIST}")
```

Comment:
I cannot pass a 'list' variable to the cmake script because lists are semicolon-separated strings in cmake and that breaks thing in linux. So you can only pass lists are strings and reconvert to list type inside cmake.